src/screens/Login.js

```diff
@@ -28,11 +28,21 @@ export default class Login extends Component {
         }
     }
 
+    static navigationOptions = () => ({
+        headerStyle: {
+          backgroundColor: '#808080',
+          borderBottomWidth: 0,
+        },
+        headerTintColor: '#fff'
+      });
+  
+
 	login = async () => {
         try {
-            const response = await login(this.state.email, this.state.password)
-            await AsyncStorage.setItem("userToken", response.token)
-        } catch(e ){
+            const response = await login(this.state.email, this.state.password);
+            await AsyncStorage.setItem("userToken", response.token);
+            this.props.navigation.navigate('AppStack');
```

Comment:
I remember you mentioning something about this AppStack navigation flow last night. Can you explain what it is to me?

Reply:
Followed the advice here: https://reactnavigation.org/docs/en/auth-flow.html

Basically theres two navigation stacks, Auth and App, they cannot talk to each other. In AuthLoader we're checking for a userToken, if it exists we send to the AppStack which for now just has the Dashboard (see Router.js) else we send to the AuthStack